report_py3o/models/py3o_server.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class Py3oServer(models.Model):
+    _name = 'py3o.server'
+
+    url = fields.Char("URL", required=True)
+    is_active = fields.Boolean("Active", default=True)
```

Comment:
Odoo convention for this field name is `active`, which will provide some special handling such as auto-hiding when deactivated